Bootstrap script SUSE 15 support

wadells opened the pull request:
## Description
This PR includes necessary changes to run robotest against SUSE 15 sp1.

Although gravity doesn't support this distribution yet, this work will come in handy when we want to add SUSE 15 support to Gravity (and thus validate it with robotest).

See the individual commit messages for more context on the two changes.

### Risk Profile
 - New feature or internal change (minor release) <!-- A non-API-breaking change which adds new functionality or improves Robotest's code without fixing a bug. -->

### Related Issues
Contributes to #248 and gravitational/gravity#1582.

## Testing Done
Logs [here](https://console.cloud.google.com/logs/viewer?authuser=0&expandAll=false&project=kubeadm-167321&minLogLevel=0&timestamp=2020-06-15T18:25:05.442000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22project%22%0Alabels.__uuid__%3D%2227d33712-3515-48a6-835e-c51dbf8d8091%22%0Alabels.__suite__%3D%22afa7468d-fd1b-4f43-8ace-206e2f28d4dd%22&dateRangeStart=2020-06-15T17:25:14.361Z&dateRangeEnd=2020-06-15T18:25:14.361Z&interval=PT1H&scrollTimestamp=2020-06-15T18:24:49.763516164Z). This run failed, but it is an expected failure:

```
The following pre-flight checks failed: [×] sles 15.1 is not supported
```

I will get a SUSE 12 run (to make sure it didn't regress) before merging.

wadells commented on assets/terraform/gce/bootstrap/suse.sh:
```diff
@@ -96,24 +111,31 @@ fi
-  df -h
+  local root_fs_size_bytes=$(sudo blockdev --getsize64 $root_fs_dev)
+  local root_fs_size_gb=$((root_fs_size_bytes / (1024 * 1024 * 1024)))
+  local expected=60
```
This should probably be tied to https://github.com/gravitational/robotest/blob/8e83284206d4acd004b50cdf0812034a14c61585/assets/terraform/gce/node.tf#L108 instead of having both as magic numbers

wadells replied:
Addressed in 7fb57ae.

a-palchikov commented on assets/terraform/gce/bootstrap/suse.sh:
```diff
@@ -96,24 +111,31 @@ fi
-  echo "Filesystem utilization after grow-root-fs:"
-  df -h
+  local root_fs_size_bytes=$(sudo blockdev --getsize64 $root_fs_dev)
+  local root_fs_size_gb=$((root_fs_size_bytes / (1024 * 1024 * 1024)))
```
nit: I believe bash support bitwise operations, so this could be:
```
$((root_fs_size_bytes >> 30))
```

wadells replied:
Addressed in 7fb57ae.

wadells commented:
Testing after addressing feedback: [logs](https://console.cloud.google.com/logs/viewer?advancedFilter=resource.type%3D%22project%22%0Alabels.__uuid__%3D%22bbc839cd-3af0-4d6f-b0eb-955dcbe3f089%22%0Alabels.__suite__%3D%22c6a2c3bd-f5a4-4084-9bf0-acdefde94fea%22%0Aseverity%3E%3DINFO&authuser=0&expandAll=false&project=kubeadm-167321)
